nifi-nar-bundles/nifi-extension-utils/nifi-hadoop-utils/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java

```diff
@@ -158,6 +158,7 @@ public abstract class AbstractHadoopProcessor extends AbstractProcessor implemen
 
 
     public static final String ABSOLUTE_HDFS_PATH_ATTRIBUTE = "absolute.hdfs.path";
+    public static final String QUALIFIED_PATH_ATTRIBUTE = "qualified.path";
```

Comment:
@mark-bathori Thanks for adding this attribute to HDFS processors!
The attribute name does not have a reference to hadoop/hdfs which would be useful in the downstream flow to identify the origin of the attribute.
As the file can be located on S3/ADLS too, I would use 'hadoop' over 'hdfs'. To emphasize that it is a URL, I would also suggest 'url' instead of 'path'. Something like: `hadoop.file.url`